### ISSUE TITLE
Could you bring us back the "Save model.bim backup here" function

### ISSUE THREAD

**Issue by Stewwe1** [TabularEditor/TabularEditor3#37]
![image](https://user-images.githubusercontent.com/49725984/120778301-78bccb80-c526-11eb-8a0f-4c067cb48355.png)
Hello Daniel,

this functionality was so amazing for us. Is there a chance to get it back? :-)

Thank you 

Stewwe

**Comment by otykier** [comment]
Sure! Adding it to the backlog.

**Comment by otykier** [comment]
In fact, this functionality is available, although it hasn't gotten it's own UI element in the Preferences dialog.

So as a workaround, until I get around to adding it to the dialog, you can simply edit the `%localappdata%\TabularEditor3\Preferences.json` file, adding the following near the top of the document (notice the double backslashes):

```json
{
    "BackupLocation": "D:\\SOURCES\\STREE\\AS\\PROD\\BIM Backup",
    ...
}
```

**Comment by otykier** [comment]
This option is available with [3.0.8](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.0.8) under Tools > Preferences > Model Deployment.
